### PR TITLE
Support VPA pre processor

### PR DESCRIPTION
**Pull request by jbartosik**

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go [hunk 1]
```diff
@@ -34,15 +34,20 @@ import (
...
 type AdmissionServer struct {
 	recommendationProvider RecommendationProvider
 	podPreProcessor        PodPreProcessor
+	vpaPreProcessor        vpaPreProcessor
 }
 
 // NewAdmissionServer constructs new AdmissionServer
-func NewAdmissionServer(recommendationProvider RecommendationProvider, podPreProcessor PodPreProcessor) *AdmissionServer {
-	return &AdmissionServer{recommendationProvider, podPreProcessor}
+func NewAdmissionServer(recommendationProvider RecommendationProvider, podPreProcessor PodPreProcessor, vpPreProcessor vpaPreProcessor) *AdmissionServer {
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: typo: vpaPreProcessor

File: vertical-pod-autoscaler/pkg/admission-controller/logic/vpa_pre_processor.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+limitations under the License.
+*/
+
+package logic
+
+import (
+	vpa_types "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2"
+)
+
+// noopVpaPreProcessor leaves pods unchanged when processing
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/logic/vpa_pre_processor.go]: leaves vpas unchanged

Comment (jbartosik) [reply vertical-pod-autoscaler/pkg/admission-controller/logic/vpa_pre_processor.go]: Done.

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go (hunk 1 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: Done.

**Pull request merged**